### ISSUE TITLE
Undefined is not an object

### ISSUE THREAD

**Issue by wheycool** [corymsmith/react-native-icons#77]
I just updated to the newest version of react-native-icons, and am running react-native v 0.8.0.

Code that was previously working is no longer. Specifically, TouchableHighlights of the icons present an error, whereas a `<Text>` equivalent works.

Here is the error:

```
undefined is not an object (evaluating 'this.refs[ICON_REF].setNativeProps')

setNativeProps
SMXIconImage.ios.js:39

_showUnderlay
TouchableHighlight.js:185

...
```

and here is my code:

```
            <View style={styles.footerNav}>
            <TouchableHighlight onPress={this.goToCamera}>  
              <Icon
                name='fontawesome|camera'
                size={32}
                color='white'
                style={styles.icon} 
                />
            </TouchableHighlight>
            <TouchableHighlight onPress={this.goToSearch}>  
              <Icon
                name='fontawesome|plus'
                size={40}
                color='white'
                style={styles.icon} 
                />
            </TouchableHighlight>
            <TouchableHighlight onPress={this.goToAddWeight}>  
              <Icon
                name='fontawesome|balance-scale'
                size={38}
                color='white'
                style={styles.icon} 
                />
            </TouchableHighlight>
            </View>
```

Any ideas? Thanks a lot!

**Comment by corymsmith** [comment]
@wheycool I've just fixed this and published v0.3.4 to NPM

**Comment by wheycool** [comment]
Nice work – thanks a lot!
